server/engine/src/main/java/org/eclipse/lsp/cobol/service/settings/ConfigHelper.java

```diff
@@ -117,6 +118,23 @@ public boolean parseCicsTranslatorOption(JsonElement options) {
     }
   }
 
+  /**
+   * Parse SQL Processing Enabled configurations to {@link SqlProcessing}
+   *
+   * @param options SQL Processing checkbox state from configuration
+   * @return Enabled if checked or Disabled otherwise, Enabled in the case of an invalid state
+   */
+  public SqlProcessing parseSQLProcessingEnabled(JsonElement options) {
+    if (!(options instanceof JsonNull)) {
+      if (options.isJsonPrimitive()) {
+        if (options.getAsJsonPrimitive().isBoolean()) {
+          return options.getAsBoolean() ? SqlProcessing.ENABLED : SqlProcessing.DISABLED;
+        }
+      }
+    }
```

Comment:
```suggestion
    if (options.isJsonPrimitive() && options.getAsJsonPrimitive().isBoolean()) {
      return options.getAsBoolean() ? SqlProcessing.ENABLED : SqlProcessing.DISABLED;
    }
```
unless you want to add logging and in that case append
```
    if (!options.isJsonNull()) {
      LOG.error(...);
    }
```